requirements/extra.txt

```diff
@@ -8,3 +8,4 @@ scikit-learn>=0.22.2
 torchtext>=0.3.1, <0.7  # TODO: temporary fix fix for compatibility
 onnx>=1.7.0
 onnxruntime>=1.3.0
+https://github.com/facebookresearch/fairscale/archive/8e85ce8c93569017521d92ceb78dba2c57c955a0.zip # TODO temporary fix till release version
```

Comment:
I think we will have to make a fork and merge sendrecv with master, so both our PR works.

Reply:
I think you should handle that within the pipe PR, just override this requirement once merged